packages/hooks/src/__tests__/useHasP2PSupportedCurrencies.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { APIProvider, useFetch } from '@deriv/api';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useHasP2PSupportedCurrencies from '../useHasP2PSupportedCurrencies';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn(),
```

Comment:
```suggestion
    useFetch: jest.fn(() => ({ data: { website_status: { p2p_config: { supported_currencies: ['usd'] } } } })),
```
@farhan-nurzi-deriv If you don't need to change the response in each test case then you don't need to create a `jest.MockedFunction`.